### DISCUSSION TITLE
Support Sphinx v6

### DISCUSSION THREAD

**Discussion by marxin** [pradyunsg/furo#556] [Potential Issue] [answered]
The current Sphinx master branch drops JS libraries:
https://www.sphinx-doc.org/en/master/changes.html#incompatible-changes

Please port `Furo`.

**Comment by pradyunsg** [comment, marked as answer]
Furo doesn't use any of the frameworks that are being dropped.

I'm literally one of the folks who advocated for the removal *because* of that and was the reviewer on multiple PRs related to the removal. :)

**Reply by marxin** [reply]
Oh great, so can I create a pull request where I remove the limitation `sphinx >= 4.0,<6.0` ?

**Reply by marxin** [reply]
Done in #560.
